test/integration/MWethOwnerWrapperIntegration.t.sol

```diff
@@ -0,0 +1,463 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+import "@forge-std/Test.sol";
+
+import {IERC20} from "@openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import {ITransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import {ProxyAdmin} from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+import {WETH9} from "@protocol/router/IWETH.sol";
+import {MTokenInterface} from "@protocol/MTokenInterfaces.sol";
+import {MErc20} from "@protocol/MErc20.sol";
+import {MWethOwnerWrapper} from "@protocol/MWethOwnerWrapper.sol";
+
+import {PostProposalCheck} from "@test/integration/PostProposalCheck.sol";
+import {BASE_FORK_ID} from "@utils/ChainIds.sol";
+
+/// @notice Integration tests for MWethOwnerWrapper on Base fork
+/// @dev Tests the wrapper deployed by MIP-B53 after proposal execution
+contract MWethOwnerWrapperIntegrationTest is PostProposalCheck {
+    MWethOwnerWrapper public wrapper;
+
+    address public temporalGovernor;
+    address public moonwellWeth;
+    address public weth;
+    address public wethUnwrapper;
+    address public mrdProxyAdmin;
+
+    // Test addresses
+    address public constant TEST_USER = address(0x1234);
+    address public constant RECIPIENT = address(0x5678);
+
+    event EthWrapped(uint256 amount);
+    event TokenWithdrawn(
+        address indexed token,
+        address indexed to,
+        uint256 amount
+    );
+
+    function setUp() public override {
+        // Run all proposals in development, including MIP-B53
+        super.setUp();
+
+        vm.selectFork(BASE_FORK_ID);
+
+        // Get deployed contract addresses
+        temporalGovernor = addresses.getAddress("TEMPORAL_GOVERNOR");
+        moonwellWeth = addresses.getAddress("MOONWELL_WETH");
+        weth = addresses.getAddress("WETH");
+        wethUnwrapper = addresses.getAddress("WETH_UNWRAPPER");
+        mrdProxyAdmin = addresses.getAddress("MRD_PROXY_ADMIN");
+
+        // Get the wrapper deployed by MIP-B53
+        address wrapperAddress = addresses.getAddress("MWETH_OWNER_WRAPPER");
+        wrapper = MWethOwnerWrapper(payable(wrapperAddress));
+    }
+
+    // ============================================
+    // Deployment and Configuration Tests
+    // ============================================
+
+    function testDeploymentOnBaseFork() public view {
```

Comment:
following up on my last question, this function name makes me think we need to test the deployment on OP fork as well.